CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Apache Pekko Releases
+
+The [Release Notes](https://pekko.apache.org/docs/pekko/current/release-notes/index.html) are available on the Apache Pekko web site.
+
+## GitHub Compare
+
+GitHub has a feature allowing you to compare 2 Git tags and see the PRs and overall code diff. You can take the example URL below and tweak it with the version tags that you want to compare.
+
+https://github.com/apache/incubator-pekko/compare/v1.0.1...v1.0.2
```

Comment:
I think a better way is we creating milestones for every release.

Reply:
this is an exact copy of what I've put on other repos - we have release notes and I'm pointing to them 

I am a volunteer. I will not spend time creating multiple copies of release notes.

Reply:
Let's merge this first then, as any updates can come up after